plugins/secom_cl.py

```diff
@@ -457,8 +457,8 @@ def _updateROIAndPixelSize(self, roi, pxs):
             _, rep, pxs = self._updateROIAndPixelSize((0, 0, 1, 1), pxs)
             return roi, rep, pxs
 
-        # compute the ROI.
-        roi = self._fitROI(roi)
+        # compute the ROI: disabled, as we want to keep the ROI independent of the repetition
+        # roi = self._fitROI(roi)
```

Comment:
This function is also called on line `485` after first calling `_adaptROI` and also finding `rep` which is compatible to the hardware. Maybe instead of commenting out it is safe to remove?

Reply:
Indeed, that was a little bit rushed. I've now replaced it by a comment explaining why we don't do that (while the standard version of this function does it).